docs/reference/search/request/stored-fields.asciidoc

```diff
@@ -40,13 +40,9 @@ If the requested fields are not stored (`store` mapping set to `false`), they wi
 
 Stored field values fetched from the document itself are always returned as an array. On the contrary, metadata fields like `_routing` are never returned as an array.
 
-Also only leaf fields can be returned via the `field` option. So object fields can't be returned and such requests
+Also only leaf fields can be returned via the `stored_fields` option. So object fields can't be returned and such requests
```

Comment:
> object fields can't be returned and such requests will fail

Looks like all object fields are just ignored, but a request itself doesn't fail

Reply:
Thanks, I pushed a commit to fix this.